src/main/webapp/resources/views/user/UserPage.html

```diff
@@ -0,0 +1,338 @@
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            <div class="panel panel-success" style="border:0">
+                <div class="panel-heading" style="background: #f5f5f5 ">
+                    <a class="btn btn-edit pull-right" href="EditUserPage.html">
+                        <span class="glyphicon glyphicon-pencil"></span></a>
+                    <h4>User Profile</h4>
+                </div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-lg-4 col-md-4 col-sm-4  hidden-xs">
+                            <img class="img-responsive img-rounded big-photo"
+                                 ng-src="http://vev.ru/uploads/images/00/07/64/2011/10/12/c26b1b.jpg" alt=""/>
+                        </div>
+                        <div class="col-lg-8 col-md-8 col-sm-8 col-xs-12">
+                            <div class="panel-info">
+                                <div class="panel-body">
+                                </div>
+                                <table class="table">
+                                    <tr>
+                                        <th>First name:</th>
+                                        <td>User`s First Name</td>
+                                    </tr>
+                                    <tr>
+                                        <th>Last name:</th>
+                                        <td>User`s Last Name</td>
+                                    </tr>
+                                    <tr>
+                                        <th>Email:</th>
+                                        <td>User`s email</td>
+                                    </tr>
+                                    <tr>
+                                        <th>Phone:</th>
+                                        <td>User`s phone</td>
+                                    </tr>
+                                    <tr>
+                                        <th>Address:</th>
+                                        <td>User`s address</td>
+                                    </tr>
+                                    <tr>
+                                        <th></th>
+                                        <td></td>
+                                    </tr>
+                                </table>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="container">
+    <div class="row">
+        <div class="col-xs-12">
+            <a href ng-click="flag=!flag" class="btn btn-success btn-block">Favourites
+                <span  class="fa" ng-class="{'fa-sort-desc' : isShowUp(flag),
+                                 'fa-sort-asc' : isShowDown(flag)}"></span></a>
+            <!--<span class="fa" ng-class="{'fa-sort-desc' : 'favourites',-->
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)